recipes/pynvml/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pynvml" %}
+{% set version = '8.0.1' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{version}}
+
+source:
+  url: https://github.com/gpuopenanalytics/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ff47e1808deda5c5f053d9cab04bade36ffd8e8e9707357dcb0dfbff98d60474
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<35]
```

Comment:
Could this be `noarch: python`? The wheel on PyPI looks like it has no binary dependencies.